test/integration/integration_admin_test.cc

```diff
@@ -53,6 +53,55 @@ TEST_P(IntegrationAdminTest, HealthCheck) {
   EXPECT_EQ("200", response->headers().Status()->value().getStringView());
 }
 
+TEST_P(IntegrationAdminTest, HealthCheckWithoutServerStats) {
+  envoy::config::metrics::v2::StatsMatcher stats_matcher;
+  stats_matcher.mutable_exclusion_list()->add_patterns()->set_prefix("server.");
+  initialize(stats_matcher);
+
+  BufferingStreamDecoderPtr response = IntegrationUtil::makeSingleRequest(
+      lookupPort("http"), "POST", "/healthcheck", "", downstreamProtocol(), version_);
+  EXPECT_TRUE(response->complete());
+  EXPECT_EQ("200", response->headers().Status()->value().getStringView());
+
+  response = IntegrationUtil::makeSingleRequest(lookupPort("admin"), "GET", "/stats", "",
+                                                downstreamProtocol(), version_);
+  EXPECT_TRUE(response->complete());
+  EXPECT_EQ("200", response->headers().Status()->value().getStringView());
+  EXPECT_THAT(response->body(), testing::Not(testing::HasSubstr("server.")));
```

Comment:
nit: it's common practice to use `using` nicknames in your _test.cc for the testing primitives as it makes it read more cleanly.

Reply:
Addressed in e86dae0.